client/api_bucket.go

```diff
@@ -57,66 +57,6 @@ type IBucketClient interface {
 	ListBucketsByPaymentAccount(ctx context.Context, paymentAccount string, opts types.ListBucketsByPaymentAccountOptions) (types.ListBucketsByPaymentAccountResult, error)
 }
 
-// GetCreateBucketApproval - Send create bucket approval request to SP and returns the signature info for the approval of preCreating resources.
-//
-// - ctx: Context variables for the current API call.
-//
-// - createBucketMsg: The msg of create bucket which defined by the greenfield chain.
-//
-// - ret1: The msg of create bucket which contain the approval signature from the storage provider
-//
-// - ret2: Return error when get approval failed, otherwise return nil.
-func (c *Client) GetCreateBucketApproval(ctx context.Context, createBucketMsg *storageTypes.MsgCreateBucket) (*storageTypes.MsgCreateBucket, error) {
```

Comment:
I don't think we need to remove those old "getApproval" methods because we need make sure the change is backward compatible.

Reply:
This is an exported method.  Clients code might have already used it.

Reply:
Yes, we should not delete this part for now.

Reply:
fixed